README.md

```diff
@@ -31,6 +31,23 @@ Or install it yourself as:
 
     $ gem install dep-selector-libgecode
 
+## Installation on Windows
+
```

Comment:
We should have a note here that ChefDK supports windows and that's the easiest way to get up and running.